apollo-router/src/configuration/snapshots/apollo_router__configuration__tests__schema_generation.snap

```diff
@@ -3518,6 +3523,24 @@ expression: "&schema"
       },
       "type": "object"
     },
+    "InvalidationEndpointConfig": {
+      "additionalProperties": false,
+      "properties": {
+        "listen": {
+          "$ref": "#/definitions/ListenAddr",
+          "description": "#/definitions/ListenAddr"
+        },
+        "path": {
+          "description": "Specify on which path you want to listen for invalidation endpoint. (default: /invalidation)",
+          "type": "string"
+        }
+      },
+      "required": [
+        "listen",
+        "path"
```

Comment:
The path description says it has a default value, but it's required at the same time.
Maybe the default should be documented here:
https://github.com/apollographql/router/pull/5614/files#diff-ea34c7fa4e3f6984aae7f7e0d20d22930d5386763c6be9d9fb2f017adf16f746R5601

Reply:
I removed the comment and default implementation